blocks/form/file.js

```diff
@@ -0,0 +1,63 @@
+function isFileAllowed(file, allowedTypes) {
+  if (!file) {
+    return false;
+  }
+  const fileExtension = file.name.split('.').pop().toLowerCase();
+  const fileType = file.type;
+  return !allowedTypes || allowedTypes.includes(fileType) || allowedTypes.includes(fileExtension);
+}
+
+function getFileList(files) {
+  const dataTransfer = new DataTransfer();
+  files.forEach((file) => dataTransfer.items.add(file));
+  return dataTransfer.files;
+}
+
+export default async function decorate(element) {
+  const wrappers = element.querySelectorAll('.form-browse');
```

Comment:
Lets add consistency behaviour for all file input. Contact us doesn't have drop box

Reply:
@jalagari you want me to add the dropbox in submit a tech support ticket form ??